portal-survey-2023.Rmd

```diff
@@ -0,0 +1,261 @@
+
+```{r fig-sharing-annotation, fig.cap="Caption", layout="l-body-outset", fig.width=9, fig.height=7}
+annotation <- tidy_survey_ordinal %>%
+  filter(area == "ability to annotate") %>%
+  mutate(response = factor(response, levels = c("Made it impossible to annotate", "Made it hard to annotate", "This was not an issue", "Not applicable"))) %>%
+   group_by(issue) %>%
+  count(response)
+
+grouped_bar(annotation)
+```
+
+
+### Overall comparison
+
+All issues except for metadata completeness had users who thought it made it "impossible" to achieve data discovery, sharing, or reuse.
+That means that there are some users being left behind in all of these areas.
+
+Users were also asked to provide a rating to summarize their overall experience in each area.
+When comparing the response distribution for these different areas all together, we see somewhat interestingly that the responses tend to be bi-modal (Figure \@ref(fig:fig-all-distributions)).
+Users who respond appear to be really pretty dissatisfied or pretty satisfied.
+While there is no area that is truly terrible, data sharing does have the lowest overall rating, with metadata sharing through the Data Curator App does having the lowest experience rating of 3 (Table \@ref(tab:table-ratings)).
+
+To improve the data sharing experience, the analysis has already suggested having clearer instructions and making sure they are seen by users.
+However, user comments also provide the insight that instructions become an issue because the current order/decoupling of events for data sharing is unintuitive -- data annotation should be done with data upload, not after.
+Thus it may not be really that more instructions are needed but that the process could be simpler and more intuitive.
+This would require both a technological and operational improvement to better combine data upload and annotation.
+
+Given that data sharing should be prioritized first, the next prioritization chooses between data findability versus data reuse.
+The survey results suggests that data findability affected more users.
+Both search functionality and the user interface are issues to target for data findability.
+
+
+```{r fig-all-distributions, fig.label="", fig.width=6.5, fig.height=10, echo=F}
+
+h <- tidy_survey_quant %>%
+  mutate(area = gsub("\\(.*", "", area))
+
+ggplot(h, aes(x = rating)) +
+  geom_histogram(binwidth=1) +
+  facet_wrap(area ~ ., ncol = 1) +
+  theme_light()
+```
+
+
+```{r table-ratings, fig.label="", echo=F}
+
+means <- tidy_survey_quant %>%
+  group_by(area) %>%
+  summarize(mean = round(mean(rating), 2))
+
+ gt(means)
+
+```
+
+
+
+## Other discussion
+
+Outside of the main survey, users left comments and suggestions regarding data sharing and the portal.
+One theme hinted at by these qualitative results that were not covered in the above analysis is improving data accessibility to patients.
+
+
+## Continuing research
```

Comment:
I would suggest a separate section for "study limitations" and then a different section for improvements, so it's clear

Reply:
Applied